Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "CLFFT"
+uuid = "fe1fc2a7-b730-4eae-aba2-587af0173dc9"
+authors = ["jakebolewski <jakebolewski@gmail.com>"]
+version = "0.2.0"
+
+[deps]
+BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+CLFFT_jll = "3d56f106-398e-55cb-9b6b-0dced46c254c"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+OpenCL = "08131aa3-fb12-5dee-8b74-c09406e224a2"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Primes = "27ebfcd6-29c5-5fa9-bf4b-fb8fc14df3ae"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
I don't think you're using most of these packages, like `BinaryProvider`, `Compat`, `FFTW`, `Libdl`, `Pkg`, `Test`.

Reply:
Fixed. I removed `BinaryProvider`, `Compat`, `Pkg` and moved `Test`, `FFTW` to `test/Project.toml`